src/filtrering/filtrering-veileder-grupper/veiledergrupper-innhold.tsx

```diff
@@ -52,7 +53,7 @@ function VeilederGruppeInnhold(props: VeilederGruppeInnholdProps) {
             {}, {gruppeId: gruppeId, sideNavn: finnSideNavn()});
         const filterVerdi = finnVeilederGruppe(gruppeId);
         setValgtGruppe(filterVerdi);
-        filterVerdi && dispatch(endreFiltervalg('veiledere', filterVerdi.filterValg.veiledere, props.filtergruppe));
+        filterVerdi && dispatch(endreFiltervalg('veilederOversikt', filterVerdi.filterValg.veiledere, props.filtergruppe));
```

Comment:
first argument is filterId, and probably its mistake if we rename it

Reply:
damn, good catch!